.github/workflows/test_clients.yaml

```diff
@@ -78,3 +80,44 @@ jobs:
         env:
           PROTOC_PATH: ${{ github.workspace }}/tmp/protobuf/bin/protoc
           DATA_SOURCE_PATH: ${{ github.workspace }}/${{ env.DATA_SOURCE_CHECKOUT_PATH }}
+
+  golang:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    env:
+      # Library & Data Source repos using side-by-side checkout.
+      LIBRARY_CHECKOUT_PATH: library # sync with defaults.run.working-directory
+      DATA_SOURCE_CHECKOUT_PATH: protos-source-repo
+    
+    steps:
+    - name: Go Library > Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        repository: googleapis/google-cloudevents-go
+        path: ${{ env.LIBRARY_CHECKOUT_PATH }}
+    
+    - name: Proto Schemas > Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        path: ${{ env.DATA_SOURCE_CHECKOUT_PATH }}
+
+    - name: Setup Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19.x
+
+    - name: Set Environment
+      run: echo "GENERATE_DATA_SOURCE=${PWD}/${DATA_SOURCE_CHECKOUT_PATH}" | tee -a $GITHUB_ENV
```

Comment:
What is PWD referring to?

Reply:
It's a standard variable that is loaded with the current working directory. The validation tests execution require this variable to be an absolute path. Feedback from @muncus in [google-cloudevents-go ](https://github.com/googleapis/google-cloudevents-go/pull/151) was to avoid the extra complication of `realpath` and use simpler string concatenation.